### ISSUE TITLE
Fullscreen in accessibility mode

### ISSUE THREAD

**Issue by jesmrec** [owncloud/ios-app#971]
Detected during regression test of version 11.6. Delayed, since it will need some research in advance to be fixed.

### Steps to reproduce

1. Enable VoiceOver in the device Settings
2. Open the app and download a previewable file (PDF, txt...)
3. Enable presentation mode -> file will be displayed in full screen mode
4. Try to dismiss full screen 

### Expected behavior
Full screen is dismissible somehow

### Actual behavior
Full screen is not posible to dismiss

iPhoneXR v14.4

**Comment by michaelstingl** [comment]
@jesmrec what do other apps do in similar cases?

**Comment by jesmrec** [comment]
Checking how full screen mode is managed in accessibility mode

## How iOS oC apps behaves

.doc file (MS word): parts of the document (or its background) can be selected and used to dismiss the full screen mode. 
.ppt file:  parts of the document (or its background) can be selected and used to dismiss the full screen mode. This is:

![Screenshot 2021-06-08 at 10 03 32](https://user-images.githubusercontent.com/14894746/121147051-d14eda00-c840-11eb-8be9-0293fbdfc268.png)

.txt: no posible to dismiss the accessibility mode (as described above)
.pdf: not posible to dismiss the accessibility mode (as described above)

## How other apps behave

**Files App:** By tapping once on the opened file, the full screen mode is on. In the screen, an indicator about the file position is displayed. By clicking on the indicator, the full screen mode can be dismissed:

![Screen Shot 2021-06-07 at 14 42 27](https://user-images.githubusercontent.com/14894746/121018417-b3309d80-c79e-11eb-96e8-3d0784bf70f6.png)

**Ms Word**: There is a "mobile view" available. Not exactly a full screen mode, but pretty similar. As well as Files App, it shows some controls that can be handled easily in accessibility mode:

![Screenshot 2021-06-08 at 08 39 19](https://user-images.githubusercontent.com/14894746/121135911-4d432500-c835-11eb-8677-36025d87dbca.png)

**BOX**: Full screen only for PDF files by tapping once (normal mode). In accessibility mode, it is not available. When the PDF file is in full screen, and then, we switch to accessibility mode, there is no way to dismiss it (same problem as oC). But, this specific scenario is not realistic. Not helpful.

**Dropbox**: Full screen mode is enabled by tapping once in normal mode, by tapping twice in accessibility mode. Same action to disable it.

**Google Docs**: Neither presentation mode nor full screen. When scrolling down, the top bar vanishes, and appears again by scrolling up. No other controls available. No help from here.

**Pages**: it offers presentation mode without full screen. Only a `Done` button in the upper bar. This app does not work fine at all in accessibility mode, difficulty to scroll down and other UX bad issues. Not a good example. 

![Screenshot 2021-06-08 at 08 31 16](https://user-images.githubusercontent.com/14894746/121134638-ecffb380-c833-11eb-8d21-8a0bb63d85d4.png)

So, as sum up, the ways other apps use to dismiss full screen in accessibility:

- Showing a page indicator (similar to the one we use in PDF viewer)
- Tapping twice or similar
- Showing controls also in full mode 
- Showing a `Done` button
- Scrolling down / Scrolling up (i don't like this one)
